devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -18,14 +18,14 @@
 import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 import physicalgraph.zigbee.zcl.DataType
 metadata {
-	definition(name: "Orvibo Motion Sensor", namespace: "smartthings", author: "SmartThings", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, mnmn: "SmartThings", vid: "generic-motion-2") {
+	definition(name: "zigbee-motion-detector", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: true, mnmn: "SmartThings", vid: "generic-motion-2") {
```

Comment:
Our naming convention is to call this `Zigbee Motion Detector`. In the past the `B` would be capitalized but the Zigbee Alliance changed the branding rules recently so now the `b` is lowercase.

Reply:
`minHubCoreVersion` should be `000.018.0000` instead as this wouldn't be supported on 17.